Zero-Gravity-Store-React/src/Components/Loader/Loader.js

```diff
@@ -0,0 +1,11 @@
+import { useState } from "react";
+import {BounceLoader, CircleLoader, ClipLoader, FadeLoader, HashLoader, MoonLoader, PropagateLoader, PuffLoader, PulseLoader, RingLoader, RiseLoader, RotateLoader, SyncLoader} from "react-spinners";
```

Comment:
Please import only used loader and you can remove all the unused ones.

Reply:
Ok sure